packaging/debian-sid/snapd.postrm

```diff
@@ -19,11 +19,13 @@ systemctl_stop() {
 }
 
 if [ "$1" = "purge" ]; then
-    # undo any bind mount to /snap that resulted from LP:#1668659
-    # (that bug can't happen in trusty -- and doing this would mess up snap.mount.service there)
-    if grep -q "/snap /snap" /proc/self/mountinfo; then
-        umount -l /snap || true
-    fi
+    # undo any bind mounts to ${SNAP_MOUNT_DIR} or /var/snap done by parallel
+    # installs or LP:#1668659
+    for mp in /snap /var/snap; do
+        if grep -q "$mp $mp" /proc/self/mountinfo; then
```

Comment:
`grep -q " $mp $mp"` (ie, space before  the first $mp) to make sure we don't catch something like `/blah/snap /snap`.